Python/executor_cases.c.h

```diff
@@ -2735,11 +2735,16 @@
         }
 
         case INSERT: {
+            PyObject *top;
+            PyObject **stuff;
+            top = stack_pointer[-1];
+            stuff = stack_pointer - 1 - oparg;
+            stuff = stack_pointer - oparg;
```

Comment:
Oh. I think the generator doesn't handle this correctly. :-( To fix it, you could rename the output variable.